### ISSUE TITLE
[BUG] [Linter] H037 False-positive on with `key=value` in attribute

### ISSUE THREAD

**Issue by silverwind** [Riverside-Healthcare/djLint#692]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: macOS
 - Python Version: 3.11.3
 - djLint Version: 1.31.0
 - template language: golang

## Issue
`key=value` syntax inside attributes is incorrectly parsed as attribute and will raise a `H037 Duplicate attribute found. class` error.

## How To Reproduce
```html
<input class="foo" placeholder="class=bar"/>
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]
```toml
[tool.djlint]
profile="golang"
include="H037"
```

**Comment by peacememories** [comment]
I ran into this problem as well, with code like this:
```html
<a hx-select="[id='{{ original.id }}']" hx-get="{% url "translation-tool:edit-translation" id=document.id %}">Edit</a>
```

It seems the linter just checks for any keyword followed by an equals sign inside a tag, which can really easily result in false-positives.

**Comment by Eric-RemoteNC** [comment]
Here is another code to showcase the bug more easily
`<div type="a" anothertype="b"></div>`
I think it's easier to fix this part first by checking the match with an extra blank at the start?
